doc/deployment/deploy_custom.md

```diff
@@ -1,6 +1,118 @@
+
+Pachyderm also includes `pachctl deploy custom` for creating customized deployments for cloud providers or on-premises use.
+It's usually invoked with the `--dry-run` flag, 
+with the command's standard output directed into a series of customization scripts or a file for editing.
+
+This document covers using `pachctl deploy custom ... --dry-run` to create a manifest for a custom, on-premises deployment.
+We won't directly address automating your deployment by means of customization scripts,
+but we do encourage you to treat your [infrastructure as code](./on-premises.html#infrastructure-as-code).
+
+## Anatomy of a Pachyderm deployment manifest
+
+When you run the `pachctl deploy ...` command with the `--dry-run` flag,
+you're generating a JSON-encoded Kubernetes manifest in one stream to standard output. 
+That manifest itself consists of a number of smaller manifests,
+corresponding to a particular aspect of a Pachyderm deployment.
+
+Pachyderm has three sets of application components usually deployed:
+- `pachd`, the main Pachyderm pod
+- `etcd`, the administrative datastore for `pachd`
+- `dash`, the web-based enterprise ui for Pachyderm
+We'll explore all of these within this document.
+
+In general, there are two categories of manifests in the file,
+roles-and-permissions-related and application-related.
+
+## Roles-and-permissions-related manifests
+
+### ServiceAccount
+
+Usually at the top of the file, this manifest has the `kind` key set to `ServiceAccount`. 
+[ServiceAccounts](https://kubernetes.io/docs/reference/access-authn-authz/service-accounts-admin/) are a way Kubernetes can assign namespace-specific privileges to applications in a lightweight way.
+Pachyderm's service account is called, appropriately enough, `pachyderm`.
+
+### Role or ClusterRole
+
+The next manifest will usually be of `kind` `Role` or `ClusterRole`, 
+depending on whether you  used the `--local-roles` flag `pachctl deploy` command.
+
+### RoleBinding or ClusterRoleBinding
+
+This manifest binds the `Rule` or `ClusterRole` to the `ServiceAccount` created above.
+
+## Application-related
+
+### PersistentVolume
+
+If you don't use [StatefulSets](./on_premises.html#statefulsets) to deploy Pachyderm
+(that is, you don't specify `--dynamic-etcd-nodes` flag), 
+the value you specify for `--persistent-disk` will cause `pachctl` to write a manifest for creating a [`PersistentVolume`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for Pachyderm's `etcd` to use in its [`PersistentVolumeClaim`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims).
+
+### PersistentVolumeClaim
+
+This is the `PersistentVolumeClaim` that Pachyderm's `etcd` will use if you don't deploy using [StatefulSets](./on_premises.html#statefulsets).
+You'll see this manifest's name in the `etcd` Deployment manifest, below.
+
+### StorageClass
+
+If you *do* use [StatefulSets](./on_premises.html#statefulsets) to deploy Pachyderm
```

Comment:
```suggestion
If you *do* use [StatefulSets](./on_premises.html#statefulsets) to deploy Pachyderm,
```

Reply:
@svekars: I'm worried that relative links won't work properly on github.  Do we have a way to make this work without resorting to full urls?

Reply:
I've seen this issue in the past and I don't know of a good way to fix it. If you have a relative link to an html file, the link won't work on GitHub. Replacing these links with full links is not a solution either. You could try to add a footnote and give the correct relative link to the .md file at the bottom of the page, but it's an ugly solution, IMHO.

Reply:
We can add a disclaimer in the docs/index.md file that says something like: "If you read this documentation on GitHub, some links might not work. For complete documentation, see https://pachyderm.readthedocs.io."